messages/Config.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
 	"InvalidStringArrayValue": "Invalid value for property '%s' in Config for %s. Expected an array of strings: %s",
-	"OnlyStringAllowedInStringArray": "Invalid value for property '%s' in Config for %s. Expected only string items in the array: %s",
```

Comment:
This message seems insufficiently different from `InvalidStringArrayValue` to warrant being its own message, and removing it lets us simplify relevant code significantly.